packages/dashboard/src/components/app-contexts.tsx

```diff
@@ -19,6 +19,11 @@ export interface Tooltips {
   showTooltips: boolean;
 }
 
+export interface RefreshTasksTable {
+  forceRefreshTask: number;
+  setForceRefreshTask: (forceRefreshTask: number) => void;
+}
+
```

Comment:
What do you think about adding these into `AppController`, instead of creating a new context? We can name it `updateTaskQueueTable`, similar to `updateSettings` too

let me know if there's something I might have missed that would make it not feasible

Reply:
This suggestion makes total sense. 
Allow us to avoid adding another context provider in app-base.tsx
Thanks, Aaron!